### DISCUSSION TITLE
Support for Firestore emulator usage for Realtime Databases

### DISCUSSION THREAD

**Discussion by hakondh** [firebase/flutterfire#3668] [Feature request]
The FlutterFire docs [describes how to connect to emulators for Firestore](https://firebase.flutter.dev/docs/firestore/usage/#emulator-usage). Would be great to have something similar for the Realtime Database as well.

**Comment by natebot13** [comment]
I was looking for information on this just yesterday, glad there's now an issue for it since I couldn't find any working solution.

**Comment by Vijay053** [comment]
Really need this feature. It will be very helpful in development. Please prioritize this.

**Comment by Vijay053** [comment]
Following code worked for me. 

    String host = Platform.isAndroid ? 'http://10.0.2.2:9000' : 'http://localhost:9000';
    dbInstance= FirebaseDatabase(
        databaseURL: '$host?ns=YOUR_DATABASE_NAMESPACE',
    );

Database namespace can be found in firebase emulator UI.
